### ISSUE TITLE
setUpClass/teadDownClass not called by dicover with py2.6

### ISSUE THREAD

**Issue by GoogleCodeExporter** [testing-cabal/unittest-ext#79]
```
What steps will reproduce the problem?

$ cat test_demo.py
import unittest2 as unittest


class TestClass(unittest.TestCase):
    @classmethod
    def setUpClass(cls):
        cls.variable = "egg"

    def test_method(self):
        self.assertEqual(self.variable, "egg")
$ discover -v
test_method (test_demo.TestClass) ... ERROR

======================================================================
ERROR: test_method (test_demo.TestClass)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/opt/stack/new/demo/test_demo.py", line 10, in test_method
    self.assertEqual(self.variable, "egg")
AttributeError: 'TestClass' object has no attribute 'variable'

----------------------------------------------------------------------
Ran 1 test in 0.000s

FAILED (errors=1)

What is the expected output? What do you see instead?

I am expecting successfully test run, but I see the tests are 
failing because the setUpClass is not called before the test case.

What version of the product are you using? On what operating system?
discover 0.4.0
RHEL 6 , python 2.6 

Please provide any additional information below.

I do not have the above issue when I am using Fedore 19 with python 2.7.

This is serious issue when I want to run tempest (openstack test suite),
 in parallel (testr/subunit) on RHEL6/py26.

The issue is solvable by using the unittest2.TestSuite when it is available.

Patch attached.
```

Original issue reported on code.google.com by `turu...@gmail.com` on 28 Nov 2013 at 12:37

Attachments:
- [unittest2-discover.patch](https://storage.googleapis.com/google-code-attachments/unittest-ext/issue-79/comment-0/unittest2-discover.patch)
- [test_demo.py](https://storage.googleapis.com/google-code-attachments/unittest-ext/issue-79/comment-0/test_demo.py)

**Comment by GoogleCodeExporter** [comment]
```
I believe this is fixed in 0.7.0 and up - discover itself is a different 
package which we can deprecate now.
```

Original comment by `robert.c...@gmail.com` on 3 Nov 2014 at 4:46

**Comment by GoogleCodeExporter** [comment]
```
Confirmed - works in unittest2, just start using that rather than discover.
```

Original comment by `robert.c...@gmail.com` on 4 Nov 2014 at 2:58
- Changed state: **Fixed**
